sdk/textanalytics/Azure.AI.TextAnalytics/tests/AbstractSummaryTests.cs

```diff
@@ -76,6 +77,7 @@ public AbstractSummaryTests(bool isAsync, TextAnalyticsClientOptions.ServiceVers
         private const int AbstractiveSummarizationSentenceCount = 3;
 
         [RecordedTest]
+        [RetryOnInternalServerError]
```

Comment:
Can an `IRepeatTest` be on a class? Might be better to support either so it's usage isn't so verbose.

Reply:
I wanted this too, but unfortunately, it only works with test methods. 😢